src/Features/Core/Portable/Diagnostics/InternalDiagnosticsOptionsProvider.cs

```diff
@@ -21,8 +21,6 @@ public InternalDiagnosticsOptionsProvider()
         }
 
         public ImmutableArray<IOption> Options { get; } = ImmutableArray.Create<IOption>(
-            InternalDiagnosticsOptions.CompilationEndCodeFix,
```

Comment:
Code path for this option was not exercised as we do not run compilation end actions in IDE live analysis, and don't want to do so either.